pkg/coscheduling/coscheduling.go

```diff
@@ -370,10 +385,42 @@ func responsibleForPod(pod *v1.Pod) bool {
 func (cs *Coscheduling) podGroupInfoGC() {
 	cs.podGroupInfos.Range(func(key, value interface{}) bool {
 		pgInfo := value.(*PodGroupInfo)
-		if pgInfo.deletionTimestamp != nil && pgInfo.deletionTimestamp.Add(PodGroupExpirationTime).Before(cs.clock.Now()) {
+		if pgInfo.deletionTimestamp != nil && pgInfo.deletionTimestamp.Add(cs.args.PodGroupExpirationTime).Before(cs.clock.Now()) {
 			klog.V(3).Infof("%v is out of date and has been deleted in PodGroup GC", key)
 			cs.podGroupInfos.Delete(key)
 		}
 		return true
 	})
 }
+
+// parsePluginConfig parses optional parameters for Coscheduling plugin
+func parsePluginConfig(args *runtime.Unknown, config *Config) error {
+	var params struct {
+		PermitWaitingTime      string `json: permitWaitingTime`
+		PodGroupGCInterval     string `json: podGroupGCInterval`
+		PodGroupExpirationTime string `json: podGroupExpirationTime`
+	}
+
+	if args == nil {
+		params.PermitWaitingTime = "1s"
+		params.PodGroupGCInterval = "30s"
+		params.PodGroupExpirationTime = "30m"
```

Comment:
Keeping the original default value "10m"?

Reply:
deleted it.